### DISCUSSION TITLE
Disconnect all

### DISCUSSION THREAD

**Discussion by CarlosRuiz-globalqss** [dbeaver/dbeaver#15512] [Q&A] [answered]
I'm not sure if this option already exists.

Sometimes I'm connected to 3 or 4 databases at the same time, when I want to disconnect all of them, at the moment I navigate one by one, right click and Disconnect.

It would be great if I can select an option (like some menu option "Disconnect All") to disconnect all of them immediately.

**Comment by uslss** [comment, marked as answer]
This option is already available in Database menu in Main menu
![image](https://user-images.githubusercontent.com/31996417/154273338-ca1f9b13-3fa7-43d9-b2c9-3d7efa74fe57.png)

**Comment by CarlosRuiz-globalqss** [comment]
Great!!!  Thank you for the fast answer.

Sorry for the noise   :-)
